src/main/java/net/sf/jabref/gui/preftabs/FileTab.java

```diff
@@ -285,10 +285,6 @@ public void storeSettings() {
             prefs.put(JabRefPreferences.NON_WRAPPABLE_FIELDS, nonWrappableFields.getText());
             updateSpecialFields = true;
         }
-        // If either of the two last entries were changed, run the update for special field handling:
```

Comment:
Why was this code removed? OK, I don't even get the deleted comment, but maybe the code was useful.